tests/integration/api/test_aws_integration.py

```diff
@@ -0,0 +1,99 @@
+import os
+from datadog import api as dog
+from datadog import initialize
+
+API_KEY = os.environ.get("DD_TEST_CLIENT_API_KEY", "a" * 32)
+APP_KEY = os.environ.get("DD_TEST_CLIENT_APP_KEY", "a" * 40)
+API_HOST = os.environ.get("DATADOG_HOST")
```

Comment:
Let's not duplicate these, and create a `constants.py` from which to import all the constants we need